### ISSUE TITLE
Install from…

### ISSUE THREAD

**Issue by IzzySoft** [ukanth/micopacks#6]
Quoting the Readme:

> Install from play store
>
> https://play.google.com/store/apps/details?id=dev.ukanth.micopacks

Well, 404 – Play Store doesn't know about the app. Maybe you could simply attach the APK to its corresponding release? Thanks!

**Comment by ukanth** [comment]
Fixed the link. Thanks. I will upload the apk's here going forward.

**Comment by IzzySoft** [comment]
Thanks! Could you start with the latest release already existing? If you're not opposed to it, I'd like to add your app [to my repo](https://apt.izzysoft.de/fdroid), for folks outside of Google Play (Huawei users, privacy folks). My updater would pick the APKs directly from `releases/` within 24h of your attaching them there. And if you wish, I could also help you getting it listed at [F-Droid](https://f-droid.org), where your AFWall+ already is present :smiley: (and I'm a happy user of that! Thanks!!)

**Comment by ukanth** [comment]
Yes. I would love to do that.But currently it has Google license verification integrated. May be next versions, I will remove that and start uploading here.

**Comment by IzzySoft** [comment]
Oh, yes, get it – in that case it doesn't help anyone having the APK available outside Play.

May I suggest reopening this issue until then? Two good arguments for that: 1) you have a little "reminder" sitting here, and 2) I'd get a ping when you finally close it as beeing done :smiley:

Thanks a lot!
